src/Libraries/DSOffice/Excel.cs

```diff
@@ -449,7 +460,7 @@ internal WorkSheet WriteData(int startRow, int startColumn, object[][] data)
 
             var c1 = (Range)ws.Cells[startRow + 1, startColumn + 1];
             var c2 = (Range)ws.Cells[startRow + numRows, startColumn + numColumns];
-            var range = ws.get_Range(c1, c2);
+            var range = ws.Range[c1, c2];
```

Comment:
Both`get_Range` and `Range` can be used interchangeably. Resharper recommends using `Range`.